change/@fluentui-react-divider-b72313a8-0cf1-43ac-b281-3c7f00b83a90.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
This should be of type `none` since it only updates docs so it doesn't bump the package.